frontend/src/components/prompt/InputBar.tsx

```diff
@@ -23,6 +24,12 @@ const InputBar = ({ placeInput, setPlaceInput, handleOpenModal }: Props) => {
     handleOpenModal();
   };
 
+  const handleIconClick = () => {
```

Comment:
`handleSearchIconClick` name should be better

Reply:
like function name? just changing it is okay?